pkg/clusters/addons/kong/builder.go

```diff
@@ -113,6 +117,14 @@ func (b *Builder) WithProxyImage(repo, tag string) *Builder {
 	return b
 }
 
+// WithControllerImage configures the ingress controller
+// container image name and tag for the Kong proxy.
```

Comment:
I believe this will be a bit more clear for the reader. WDYT?

```suggestion
// WithControllerImage configures the ingress controller container image name and tag.
```